packages/gitbook/src/components/DocumentView/Table/table.module.css

```diff
@@ -1,3 +1,54 @@
+/* Detect whether a scrollbar exists on the table */
+@keyframes detect-scroll {
+    from,
+    to {
+        --can-scroll: ;
+    }
+}
+
+/* Apply styles to the Table if scrollbar exists */
```

Comment:
Fun trick I learned as I don't think there is any other css selector or query that can be used to select elements with a scrollbar: https://csscade.com/can-you-detect-overflow-with-css/